Codeinwp/optimole-wp
# Fix Images with a Bullet symbol not being optimized

### All Submissions:

 

### Changes proposed in this Pull Request:
 
Previously, images with UTF character 2022, ie the bullet, weren't being optimized. The fix makes sure they are working properly.

Closes https://github.com/Codeinwp/optimole-service/issues/738

### How to test the changes in this Pull Request:

1. Upload an image with the bullet character in its filename, for example `DSCF4559-Edité-•-©-Josef-Helie.png`
2. Add it to your post, and confirm the Optimole CDN URL is being applied to it.

### Other information:
